### PR TITLE
Redspace Purge

### PR DESCRIPTION
**Pull request by PopeDaveThe3th**
## What Does This PR Do
Obliterates all references to redspace within the code.

Rewrites some cult spell descriptions to account for this.

## Why It's Good For The Game
By order of the Lore Team, Redspace is to be removed in its entirety.

Redspace was never anything more than edgy bluespace, and an even larger source of handwavium.

We have more interesting plans for its replacement.

## Testing
- Loaded in
- Checked objects
- Descriptions and names are updated

## Changelog
:cl:
del: Deletes references to Redspace
tweak: Rewrites names and descriptions to account for the destruction of Redspace
/:cl:

### CODE REVIEW

File: code/modules/station_goals/bluespace_tap.dm [hunk 1]
```diff
@@ -260,7 +260,7 @@
 	. = ..()
 
 	if(length(active_nether_portals))
-		icon_state = "redspace_tap"
+		icon_state = "Cascade_tap"
```
Comment (HMBGERDO) [inline code/modules/station_goals/bluespace_tap.dm:263]: what is that supposed to be? 
there is no icon like that?
image

File: code/modules/station_goals/bluespace_tap.dm [hunk 2]
```diff
@@ -275,7 +275,7 @@
 	underlays.Cut()
 
 	if(length(active_nether_portals))
-		. += "redspace"
+		. += "abyssal"
```
Comment (HMBGERDO) [inline code/modules/station_goals/bluespace_tap.dm:278]: same here, this must be real icon

**Comment by HMBGERDO** [comment]
i guess you just need to change .dmi file `icons/obj/machines/bluespace_tap.dmi` and just rename those to your new names. You can do that with Dream Maker

File: code/datums/spells/construct_spells.dm [hunk 3]
```diff
@@ -52,7 +52,7 @@
 
 /obj/effect/proc_holder/spell/aoe/conjure/build/soulstone
 	name = "Summon Soulstone"
-	desc = "This spell reaches into Redspace, summoning one of the legendary fragments across time and space"
+	desc = "This spell uses vile sorcery to create a spirit-trapping soulstone"
```
Comment (Henri215) [inline code/datums/spells/construct_spells.dm:55]:
```suggestion
	desc = "This spell uses vile sorcery to create a spirit-trapping soulstone."
```

File: code/game/gamemodes/cult/cult_structures.dm [hunk 4]
```diff
@@ -70,7 +70,7 @@
 		to_chat(user, "[heathen_message]")
 		return
 	if(invisibility)
-		to_chat(user, "<span class='cultitalic'>The magic in [src] is being channeled into Redspace, reveal the structure first!</span>")
+		to_chat(user, "<span class='cultitalic'>The magic in [src] is being supressed, reveal the structure first!</span>")
```
Comment (Henri215) [inline code/game/gamemodes/cult/cult_structures.dm:73]:
```suggestion
		to_chat(user, "<span class='cultitalic'>The magic in [src] is being suppressed, reveal the structure first!</span>")
```

**Comment by PopeDaveThe3th** [comment]
> i guess you just need to change .dmi file `icons/obj/machines/bluespace_tap.dmi` and just rename those to your new names. You can do that with Dream Maker

Icons should be fixed now

**Review by Burzah — APPROVED** [review]

**Review by S34NW — APPROVED** [review]

**Pull request merged**